src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,14 @@
+@mixin squares () {
+  box-sizing: border-box;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: $cell_size;
+  height: $cell_size;
+  background-color: #eee;
+  cursor: pointer;
+  border: 1px solid black;
+  transition: transform 0.5s ease-in-out;
+  padding: 10px;
```

Comment:
that's all styles for your `calendar__day` elements, no need to put it all inside a mixin, you should use mixin for SOME properties, not for an entire class